_component_design/checkboxes.md

```diff
@@ -218,17 +304,16 @@ In the case of an error, always include an error message below the checkbox grou
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-#### Indeterminate
+## Indeterminate checkboxes
 
-The indeterminate state is only used with bulk selections to indicate that one or more, but not all options, have been selected. Refer to the [tables]({{site.baseurl}}/components/tables.html) page for more information.
+Use an indeterminate state with a nested group that provides for bulk selection. An indeterminate checkbox indicates that the user has selected one or more, but not all, options in a nested group. See  [tables]({{site.baseurl}}/components/tables.html) for more information about checkboxes in tables.
```

Comment:
OOOK, this still needs work. try this:

Use an indeterminate state only in table header checkbox. An indeterminate checkbox indicates that the user has selected one or more, but not all, of the child checkboxes in the table. See  [tables]({{site.baseurl}}/components/tables.html) for more information about checkboxes in tables.